nrf_security/CMakeLists.txt

```diff
@@ -107,6 +107,9 @@ else()
 
 endif()
 
+#finally adding the crypto lib
+add_subdirectory(../crypto crypto_copy)
```

Comment:
Crypto directory may already have been included here:
https://github.com/nrfconnect/sdk-nrfxlib/blob/7588c863d46d35084463e7ee871d79b4ef935f85/CMakeLists.txt#L16

so why adding this here ?
And why isn't adding the crypto folder at the normal level sufficient ?